.github/workflows/ee.yml

```diff
@@ -0,0 +1,121 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+name: execution environment
+on:
+  # Run CI against all pushes (direct commits, also merged PRs), Pull Requests
+  push:
+    branches:
+      - main
+      - stable-*
+  pull_request:
+  # Run CI once per day (at 05:30 UTC)
+  # This ensures that even if there haven't been commits that we are still testing against latest version of ansible-builder
+  schedule:
+    - cron: '30 5 * * *'
+
+env:
+  NAMESPACE: community
+  COLLECTION_NAME: sops
+
+jobs:
+  build:
+    name: Build and test EE (Ⓐ${{ matrix.runner_tag }})
+    strategy:
+      matrix:
+        runner_tag:
+          - devel
+          - stable-2.12-latest
+          - stable-2.11-latest
+        execute_playbook:
+          - ansible-playbook -v community.sops.install_localhost
+        # This currently does not work with Ansible 2.9 since the dnf module isn't available.
+        # include:
+        #   - runner_tag: stable-2.9-latest
+        #     execute_playbook: ansible localhost -m include_role -a name=community.sops.install -e sops_install_on_localhost=true
```

Comment:
The **Ansible** module `dnf` is there, but the **Python** module `dnf` the Ansible module needs isn't there - or at least not for the Python interpreter that is used to run Ansible.

Reply:
Hmm, that's weird. It looks like this runs on CentOS Stream 8. Can you try adding `-e ansible_python_interpreter=/usr/libexec/platform-python` which is the correct Python interpreter for that release?

Reply:
That might work, the system Python is 3.6.x, while Ansible is using Python 3.8.x. This probably works fine with ansible-core 2.11+ since it respawns the Ansible module with the correct Python interpreter, but 2.9 does not yet do that...

Reply:
> Ansible is using Python 3.8.x.

Yes, that would be the issue. ansible-core 2.12 (or 2.11?) implemented the "respawning" feature and made some interpreter discovery improvements, but 2.9 of course doesn't have those. 2.9 defaults to /usr/bin/python, which seems to be python3.8 in this image. python3-dnf is only available under the system Python. Setting `ansible_python_interpreter=/usr/libexec/platform-python` should fix this.

Reply:
561f443ddb85fa1dae9351d873b673ad03868320 implements this, and it works! :tada: